sklearn/metrics/_ranking.py

```diff
@@ -202,10 +211,16 @@ def _binary_uninterpolated_average_precision(
         return -np.sum(np.diff(recall) * np.array(precision)[:-1])
 
     y_type = type_of_target(y_true)
+
+    # Multiclass is a specific case of the multilabel setting here
```

Comment:
I'd say "we can reduce to the multi-label case" or "we can reuse the multi-label case"